src/common.rs

```diff
@@ -235,8 +238,11 @@ pub fn check_sig(r: &Scalar, s: &Scalar, msg: &BigInt, pk: &Point) -> bool {
     let pubkey_a = pk.get_element().serialize();
 
     let pubkey = secp256k1::PublicKey::parse(&pubkey_a).unwrap();
+
+    #[cfg(target_arch = "wasm32")]
     crate::console_log!("pubkey: {:?}", pubkey);
-    crate::console_log!("address: {:?}", public_key_address(&pubkey));
+    #[cfg(target_arch = "wasm32")]
+    crate::console_log!("address: {:?}", hex::encode(public_key_address(&pubkey)));
```

Comment:
public_key_address does not comply with https://github.com/ethereum/EIPs/blob/master/EIPS/eip-55.md, plz fix it later in another PR. @Weber-H2O2